app/templates/components/carousel-gallery.hbs

```diff
@@ -60,14 +55,9 @@
      <li data-target="#slideThree" data-slide-to="0" class="active"></li>
      <li data-target="#slideThree" data-slide-to="1"></li>
      <li data-target="#slideThree" data-slide-to="2"></li>
-     <li data-target="#slideFour" data-slide-to="3"></li>
-
    </ol>
    <div class="carousel-inner">
      <div class="carousel-item active">
-       <img class="d-block w-100" src="{{rootURL}}img/agencies/epa-Logo.jpg" alt="First slide">
-     </div>
-     <div class="carousel-item">
        <img class="d-block w-100" src="{{rootURL}}img/agencies/fda-logo.png" alt="Second slide">
      </div>
```

Comment:
alt label is off in terms of first, second, third.. not worth holding the PR up over imho... the alt labels are uninteresting anyway.  :)